components/Weather.js

```diff
@@ -33,72 +54,75 @@ export default class Weather extends React.Component {
           // See error code charts below.
           console.log(error.code, error.message);
         },
-        {enableHighAccuracy: true, timeout: 15000},
+        { enableHighAccuracy: true, timeout: 15000 },
       );
     }
   };
-  _parseWeatherData = (data) => {
+
+  parseWeatherData = (data) => {
```

Comment:
You can use destructuring here. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment
```suggestion
  parseWeatherData = ({ data: { city: {name} } }) => {
```